docs/tasks/inject-data-application/podpreset.md

```diff
@@ -4,66 +4,13 @@ approvers:
 title: Inject Information into Pods Using a PodPreset
 ---
 
-You can use a `podpreset` object to inject certain information into pods at creation
-time. This information can include secrets, volumes, volume mounts, and environment
-variables.
-
-See [PodPreset proposal](https://git.k8s.io/community/contributors/design-proposals/service-catalog/pod-preset.md) for more information.
+You can use a `podpreset` object to inject information like secrets, volume
+mounts, and environment variables etc into pods at creation time.
+This task shows some examples on using the `PodPreset` resource.
```

Comment:
Just to provide the relevant backlink, could you include:

     You can get an overview PodPresets at [Understanding Pod Presets](/docs/conceptsworkloads/pods/podpreset/)